### ISSUE TITLE
Bookings that fall under schedule

### ISSUE THREAD

**Issue by P4pomepeii** [choozexiann/BBDC-PRACTICAL-BOOKING-BOT#1]
Hey! Bot works amazing but is there a way to by pass the statement attached below? From what it seems now i can't click cancel on the pop up. Any help will do!
![image](https://user-images.githubusercontent.com/96627895/195072490-49494515-1e21-45c5-b708-5f91cd951c6f.png)

**Comment by zexianchoo** [comment]
Hey! Is this still a problem right now? Do let me know so i can improve on the bot :)

**Comment by P4pomepeii** [comment]
nope! however the bbdc completely changed the ui of the website!!

**Comment by P4pomepeii** [comment]
so far my code only manages to access to the booking page, however can't click on the practical. This is a very simplified version `from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.common.exceptions import *
from selenium.webdriver.support import expected_conditions as EC
from selenium import webdriver
import time
PATH = r"C:..."
driver = webdriver.Chrome(PATH)
wait = WebDriverWait(driver, 300)
driver.get('https://booking.bbdc.sg/#/login?redirect=%2Fhome%2Findex')
username = driver.find_element(By.ID, "input-8")
username.send_keys("..")
password = driver.find_element(By.ID, 'input-15')
password.send_keys("..")
time.sleep(2)
button = driver.find_element(By.CLASS_NAME, "v-btn__content")
button.click()
wait.until(EC.visibility_of_element_located((By.LINK_TEXT, 'Booking'))).click()
wait.until(EC.visibility_of_element_located((By.XPATH, 'v-tab'))).click()`

**Comment by zexianchoo** [comment]
damn yeah I just saw the changes on the BBDC site - I'll try to see what I can do but this looks pretty difficult haha. Thank you!

**Comment by P4pomepeii** [comment]
hello! any updates on this?
